snode-src/duckcoin.go

```diff
@@ -203,6 +205,7 @@ func mine(numOfBlocks int, data string, receiver string, amount int) {
 	}
 }
 
+// loadDifficultyAndURL loads the URL from the config file, and then loads the difficulty from that.
```

Comment:
Resolve all ambiguity
```suggestion
// loadDifficultyAndURL loads the server URL from the config file, and then loads the difficulty by contacting that server.
```